app/utils.py

```diff
@@ -107,7 +108,11 @@ def get_user_substitutions(request):
 def hackathon_vars_processor(request):
     c = get_substitutions_templates()
     c.update(get_user_substitutions(request))
-    c.update({'slack_enabled': settings.SLACK.get('token', None) and settings.SLACK.get('team', None)})
+    c.update({
+        'slack_enabled': settings.SLACK.get('token', None) and settings.SLACK.get('team', None),
+        'mentor_expires': settings.MENTOR_EXPIRES,
+        'volunteer_expires': settings.VOLUNTEER_EXPIRES,
+    })
```

Comment:
Why is this set here instead of above like the others?

Reply:
It was already like this

Reply:
I just added 2 more